hub.json

```
@@ -244,5 +244,8 @@
     ],
     "hightouchio": [
         "dbt-hightouch"
+    ],
+    "everpeace" [
```

I think this is missing a colon between the JSON key and value.
```suggestion
    "everpeace": [
```